docs/admin_api/purge_history_api.rst

```diff
@@ -10,7 +10,7 @@ paginate further back in the room from the point being purged from.
 
 The API is:
 
-``POST /_matrix/client/r0/admin/purge_history/<room_id>[/<event_id>]``
+``POST /_matrix/client/r0/admin/purge_history/<room_id>[/<event_id>]?access_token=<access_token>``
```

Comment:
this is not correct; it's better to use an `Authorization` header than a query param. In any case, it's why it says "including an ``access_token`` of a server admin" on the next line, as do all the documents in this directory. Please don't mix up changes to the official docs with your `contrib` changes.

Reply:
Where do I find a documentation about how to send an Authorization header?